### DISCUSSION TITLE
Ubuntu 23.04 support?

### DISCUSSION THREAD

**Discussion by stephenh** [regolith-linux/regolith-desktop#847] [Q&A]
Hi! I noticed that Ubuntu 23.04 is out now; usually I wait a little while to upgrade, so np/no rush, but just curious, is that something I could try, or should I definitely hold off?

I also noticed a few discussions of "Regolith 3 + sway", so was wondering if maybe Ubuntu 23.04 would start requiring sway, hence necessitate waiting for a Regolith 3 release, or something like that?

Thanks!

**Comment by kgilmer** [comment]
Hi @stephenh ,

> is that something I could try, or should I definitely hold off?

We're actively working on an alpha release of Regolith 3.0 which includes support for Ubuntu 23.04.  I would expect this alpha to be available in a week or two.  Currently undergoing some build infrastructure migrations.

If you wanted to try out Regolith 3.0 now, you can install from the `unstable` repo, however I would not use this for anything other than testing.

The unstable repo can be added by substituting this `apt` configuration line from the official installation instructions:

```console
echo deb "[arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] \
https://regolith-desktop.org/unstable-ubuntu-lunar-amd64 lunar main" | \
sudo tee /etc/apt/sources.list.d/regolith-unstable.list
```

When/if you want to remove this simply delete the file `/etc/apt/sources.list.d/regolith-unstable.list` and update `apt`.

> I also noticed a few discussions of "Regolith 3 + sway", so was wondering if maybe Ubuntu 23.04 would start requiring sway

At least until now, every new Ubuntu release has been matched with a new Regolith release.  This is because there are always fixes or additions that necessitate a new release.  The current plan is to release the Wayland variant of the Regolith session as an alpha/preview to collect user feedback.  Until the forseeable future, Regolith will always offer an X11 version of the session.  Regolith 3.0 is the first version that will offer a Wayland version based on the Sway compositor.  @SoumyaRanjanPatnaik is the Regolith contributor that has developed the Wayland session.

Happy to answer any more questions, and hope to have a more clear roadmap and release info for users in the future so there are no mysteries.

**Reply by stephenh** [reply]
That all sounds great!

I'm really curious to try the Sway side of things, but selfishly only from the comfort of a Regolith release that has already figured out all of the hard things. :-)

Thanks for the update, and yep given my comfort/expertise level, I'll sit tight and way for an official release whenever it comes out. :-)

**Reply by lawrencegripper** [reply]
This is really excited, looking forward to trying out regolith 3 (both away and x11 version)

Thanks for all the great work!

**Reply by ronebrandao** [reply]
I installed 23.04 and forgot to check if regolith was available...

Nonetheless, I'll give the unstable version a try :stuck_out_tongue_winking_eye:. Can't live without it!

**Comment by falko** [comment]
@kgilmer could you give a status update here or point out where one could track progress with the Ubuntu 23.04 support?

**Reply by kgilmer** [reply]
Hi @falko , sure

You can see project status here: https://github.com/orgs/regolith-linux/projects/26

You can see builds happening here: https://github.com/regolith-linux/voulage/actions

And looking at Github's repo list view will give you a sense of recent changes: https://github.com/regolith-linux

Regarding the project status, we're prepping the first beta release for 3.0.  Should expect an announcement soon on that.

**Comment by kgilmer** [comment]
Regolith 3.0 beta 1 has been released.  More info here: https://regolith-desktop.com/docs/reference/Releases/regolith-3.0-release-notes/

**Reply by lawrencegripper** [reply]
Nice, excited to give it a try :+1: :partying_face:

**Reply by clarkevans** [reply]
Thank you so, so much.
